### ISSUE TITLE
8149 x 3490 is returned when entering these same dimensions. 

### ISSUE THREAD

**Issue by hutber** [ryanhefner/calculate-aspect-ratio#4]
So if I input 8149x3490 then the aspect ratio that is returned is: `8149 3490`

Likewise with:
`900 712 =  225 178`

Just thought you would want to know.

**Comment by ryanhefner** [comment]
Yeah, I have been debating that since I created this package, and the site. It returns the inputs because it looks to find the smallest whole number that each number can be divided by, but in your examples there is no lesser number than the numbers themselves. The other option would be to find the smallest of the two, and then offer a float for the other value, but then you have to start dealing with how detailed do you want to get with decimal lengths and accuracy.

Like I said, I have been debating this. Do you think it should be the later solution?
